microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -6,31 +6,44 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 08/03/2021
+ms.date: 04/24/2023
 ---
 # Understand the different WebView2 SDK versions
 
-The NuGet package for the WebView2 SDK contains both a release and prerelease package.  Either use a prerelease SDK with a preview channel of Microsoft Edge, or use a release SDK with the WebView2 Runtime.
+The NuGet package for the WebView2 SDK contains both a Prerelease package and a Release package.<!--todo: true?  actually two separate NuGet packages?-->  Either use a Prerelease SDK with a preview channel of Microsoft Edge, or use a Release SDK with the WebView2 Runtime.
```

Comment:
@novac42 @peiche-jessica -- Does this sentence need to be reworded?  
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/versioning -- 

"The NuGet package for the WebView2 SDK contains both a release and prerelease package."

true?

Reply:
release and prerelease are two separate Nuget packages.

Reply:
Rewrote Sentence 1 as:
The WebView2 SDK is provided as a Prerelease or Release version of the **Microsoft.Web.WebView2** NuGet package.